lib/capybara-screenshot.rb

```diff
@@ -106,11 +106,11 @@ def self.new_saver(*args)
     end
 
     def self.after_save_html &block
-      Saver.after_save_html &block
+      Saver.after_save_html( &block )
```

Comment:
I would really prefer not to have spaces between the parenthesis and the arguments.  See https://github.com/rubocop-hq/ruby-style-guide#spaces-braces.  Would you mind changing here and elsewhere?

Reply:
Will fix the style.

As to why ward on private accessors, I am just guessing, but I think it is to force the coder to consider implications and define them explicitly. Explicit definition essentially says "yes, I thought about, and this is exactly what I want to do, and no, I did not stick my accessors into the private: block by mistake." Something like that.